src/plugin.scss

```diff
@@ -105,96 +190,30 @@ during ads and capture click events on IE<11 */
   width: 100%;
   height: 100%;
 
-  /* IE8 fallback */
+  // IE8 fallback
   background-color: #1a1a1a;
   -ms-filter: "progid:DXImageTransform.Microsoft.Alpha(Opacity=50)";
 
-  /* modern browsers */
+  // modern browsers
   background-color: rgba(0, 0, 0, 0.5);
 }
 
-/* Parametric rules. These are specialized for touch and mouse-based devices */
+// Parametric rules. These are specialized for touch and mouse-based devices
```

Comment:
Most of the CSS that was in this mixin was redundant and could be moved out. This is the main source of the file size reduction.